libecho/src/EchoRawConnectionImpl.cc

```diff
@@ -1,30 +1,30 @@
 #include "EchoRawConnectionImpl.h"
-#include "BitReceiver.h"
-#include "BitSender.h"
+#include "BitReceiverv2.h"
+#include "BitSenderv2.h"
 
-EchoRawConnectionImpl *EchoRawConnectionImpl::getBitEchoRawConnection(int winSize, int receiveLoFreq, int receiveHiFreq,
-                                                                      int sendLoFreq, int sendHiFreq, int leftLim,
-                                                                      int rightLim) {
+EchoRawConnectionImpl *EchoRawConnectionImpl::getBitEchoRawConnection(int win_size, int send_freq, int recv_freq,
```

Comment:
Try to add a new one than changing previous as we probably want to keep previous code in case it is needed idk